src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html

```diff
@@ -16,18 +16,18 @@ <h4 class="mb-0">{{ 'artemisApp.examManagement.exerciseGroups' | artemisTranslat
 <div class="mt-3 text-end" id="number-groups" style="font-size: 0.85rem">
     {{ 'artemisApp.examManagement.numberOfExerciseGroups' | artemisTranslate }} {{ exerciseGroups ? exerciseGroups.length : 0 }}
 </div>
-<div *ngFor="let exerciseGroup of exerciseGroups; let i = index" class="exercise-group mt-3">
+<div *ngFor="let exerciseGroup of exerciseGroups; let i = index" class="exercise-group mt-3" id="{{ 'group-' + i }}">
```

Comment:
Same as in the other PR, check with https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
I talked to @krusche about this and we decided that we are going to stick with the id selector for now.